tasks/joomla.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Create Joomla Directories
+  file:
+    path: "{{ item }}"
+    state: directory
+    # mode: 0755
+  with_items:
+    - "{{ docker_home }}/joomla"
+
+- name: Create MySQL Directorys for Joomla
+  file:
+    path: "{{ item }}"
+    state: directory
+    # mode: 0755
+  with_items:
+    - "{{ docker_home }}/JoomlaDB"
+
+- name: MySQL Docker Container for Joomla
+  docker_container:
+    name: mysql_joomla
+    image: mysql
```

Comment:
What version of MySQL? If a user has watchtower enabled this will constantly pull the latest available which may be undesired. Perhaps better to specify an image version tag too.

Reply:
later on i did